### PR TITLE
Trino upgrade

### PR DESCRIPTION
**Pull request by LtChae**
## Description

- Updated all apps that use prestige to use the new version that handles Trino
- Updated pipeline to use the new version of prestige
- Updated docker-compose files with the latest postgres, metastore, minio, and trino

## Reminders:

- [ ] If changing elixir code in an "app", did you update the relevant version
      in `mix.exs`?
- [ ] If altering an API endpoint, was the relevant postman collection updated?
  - [ ] If a new version of `smart_city` is being used (new fields on a struct), were the relevant postman collections updated?

### CODE REVIEW

File: apps/e2e/test/e2e_test.exs [hunk 1]
```diff
@@ -145,7 +145,6 @@ defmodule E2ETest do
       eventually(fn ->
         with resp <- HTTPoison.get!("http://localhost:4000/api/v1/organizations"),
              [org] <- Jason.decode!(resp.body) do
-          assert org["dn"] == "cn=end_to,ou=integration,#{base}"
```
[the remark this answers is not in the document]
Comment (LtChae) [reply apps/e2e/test/e2e_test.exs:148]: It was causing a test to flicker and ldap support hasn't been needed for ~3 years now.

File: apps/e2e/2cb3b3a6-59f0-4de2-8555-2dd1ae3dc866 [hunk 2]
```diff
@@ -0,0 +1 @@
+true,foobar,10
```
[the remark this answers is not in the document]
Comment (LtChae) [reply apps/e2e/2cb3b3a6-59f0-4de2-8555-2dd1ae3dc866:1]: I did not, good catch

File: apps/e2e/test/e2e_test.exs (hunk 1 above)
[the remark this answers is not in the document]
Comment (bmitchinson) [reply apps/e2e/test/e2e_test.exs:148]: Glad to see this removed, I had no idea it was related to LDAP and have noticed it flickers a ton.

File: apps/discovery_api/test/integration/docker-compose.yaml [hunk 3]
```diff
@@ -71,13 +84,18 @@ services:
...
-      test: ["CMD-SHELL", "kafka-topics.sh --zookeeper zookeeper:2181 --list | grep event-stream || exit 1"]
+      test: ["CMD-SHELL", "cat /opt/kafka/logs/server.log | grep \"Previous Leader Epoch was: -1\""]
       interval: 10s
       timeout: 20s
       retries: 3
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
```
Comment (bmitchinson) [inline apps/discovery_api/test/integration/docker-compose.yaml:98]: Looking forward to using these new images, maybe test startup time will be a bit faster.

Why was the redis healthcheck removed? As well as metastore's. Can the new images for those have healthchecks? I believe divo uses them to surface if there was issues starting them up, which has been helpful in the past.

File: apps/discovery_api/test/integration/docker-compose.yaml [hunk 4]
```diff
@@ -1,67 +1,80 @@
...
+    image: minio/minio:RELEASE.2022-07-17T15-43-14Z.fips
+    command: server /data --console-address ":9001"
+    volumes:
+      - minio:/data
+    ports:
+    - "9000:9000"
+    - "9001:9001"
+    depends_on:
+      - init
+  presto:
```
Comment (bmitchinson) [inline apps/discovery_api/test/integration/docker-compose.yaml:57]: are we keeping references to presto in the codebase? Can this image be renamed to trino?

**Review by bmitchinson — APPROVED** [review]

File: apps/discovery_api/test/integration/docker-compose.yaml (hunk 4 above)
Comment (bmitchinson) [reply apps/discovery_api/test/integration/docker-compose.yaml:57]: There are 200+ mentions of `Presto` in the codebase, including a view "Presto" services, so nevermind this 👍

**Comment by bmitchinson** [comment]
@LtChae you'll need to update the forklift mix.exs

File: apps/discovery_api/test/integration/docker-compose.yaml (hunk 3 above)
Comment (LtChae) [reply apps/discovery_api/test/integration/docker-compose.yaml:98]: Hmm, a valid question. I haven't noticed issues with startup so far. Maybe as part of a followup card assuming the builds go well?

File: apps/e2e/test/e2e_test.exs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfolk2015) [reply apps/e2e/test/e2e_test.exs:148]: I would never have guessed that was related to ldap

**Review by jfolk2015 — APPROVED** [review]

**Pull request merged**